iroh-net/src/defaults.rs

```diff
@@ -1,14 +1,19 @@
 //! Default values used in [`iroh-net`][`crate`]
 
-use url::Url;
-
-use crate::{RelayMap, RelayNode};
-
+use iroh_base::relay_map::QuicConfig;
+/// The default QUIC port used by the Relay server to accept QUIC connections
+/// for QUIC address discovery
+///
+/// The port is "QUIC" typed on a phone keypad.
+pub use iroh_base::relay_map::DEFAULT_QUIC_PORT as DEFAULT_RELAY_QUIC_PORT;
```

Comment:
😂 dig asked for the rename, since it would no longer have the `relay_map` prefix

we can also...not have this in the defaults & just import it from iroh_base where needed.

Reply:
okay actually, I just renamed them all to DEFAULT_RELAY_QUIC_PORT. This isn't like STUN that has a default port, this our personal QUIC port, so it feels like addition of `RELAY` is justified.